layouts/shortcodes/blocks/resource.html

```diff
@@ -10,7 +10,7 @@
 {{ $title := .Get "title" | default "Video title" }}
 
 <div class="col-lg-4 mb-3 mb-lg-0 text-center ">
-  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" src="{{ $thumbnail }}"></a>
+  <a href="{{ $url }}"><img class="img-fluid img-thumbnail" alt="image" src="{{ $thumbnail }}"></a>
```

Comment:
Maybe this could be the video title instead?

Reply:
I agree, thanks